apps/epic-react/src/components/video-overlays/components/progress.tsx

```diff
@@ -0,0 +1,28 @@
+'use client'
+
+import * as React from 'react'
+import * as ProgressPrimitive from '@radix-ui/react-progress'
+
+import {cn} from '@skillrecordings/ui/utils/cn'
+
+export const Progress = React.forwardRef<
+  React.ElementRef<typeof ProgressPrimitive.Root>,
+  React.ComponentPropsWithoutRef<typeof ProgressPrimitive.Root>
+>(({className, value, ...props}, ref) => (
+  <ProgressPrimitive.Root
+    ref={ref}
+    data-progress-bar=""
+    className={cn(
+      'relative h-4 w-full overflow-hidden rounded-full bg-secondary',
+      className,
+    )}
+    {...props}
+  >
+    <ProgressPrimitive.Indicator
+      data-indicator=""
+      className="h-full w-full flex-1 bg-blue-500 transition-all"
```

Comment:
`bg-primary` makes the progress bar invisible because so copied component and added `bg-blue-500`